kolibri/plugins/setup_wizard/assets/src/vue/index.vue

```diff
@@ -0,0 +1,135 @@
+<template>
+
+  <div class="device-owner-creation">
+    <div class="wrapper">
+      <img class="logo" src="./icons/logo-min.png">
+      <div class="container">
+        <div class="title">Device Owner</div>
+        <br>
+        <div class="description">To use Kolibri, there are need to be at least one device owner. This account will have the highest privilege that can configure all settings about this installation.</div>
+        <div class="creation-form">
+          <br><input :class="{ 'input-error': username_error }" type="text" v-model="username" placeholder="Username"><br>
+          <br><input :class="{ 'input-error': password_error }" type="text" v-model="password" placeholder="Password"><br>
+        </div>
+        <br>
+        <div class="title">Facility</div>
+        <br>
+        <div class="description">To use Kolibri, there are need to be at least one facility. This facility can represent your school or trainning center, and classrooms and learnning groups are orgnized under this facility.</div>
```

Comment:
"You also need to create a Facility, which represents your school, computer lab, or other location where this installation will be used."